src/js/express.js

```diff
@@ -0,0 +1,148 @@
+                        var functionName = middlewareToLoad[i];
+                        if (component[functionName]) {
+                            try {
+                                that.model.router.use(component[functionName]);
+                            }
+                            catch (e) {
+                                console.error("Error loading middleware function '" + functionName + "' in module '" + key + "':" + e);
+                            }
+                        }
+                        else {
+                            console.error("Middleware '" + functionName + "' does not exist in module '" + key + "'...");
+                        }
+                    }
+                }
+                else {
+                    console.log("Middleware module has no middleware to load, can't continue...");
+                }
+            }
+            else if (fluid.hasGrade(component.options, "gpii.express.helper")) {
+                var helpers = component.getHelpers();
+                Object.keys(helpers).forEach(function(key){
+                    if (helpersToLoad[key]) {
+                        console.error("Another module has already loaded a helper named '" + key + "'. This instance will not be enabled.");
+                    }
+                    else {
+                        helpersToLoad[key] = helpers[key];
+                    }
+                });
+            }
+        });
+    }
+
+    if (that.options.config.express.views) {
+        var viewRoot = that.options.config.express.views;
+        var handlebarsConfig = {
+            defaultLayout: "main",
+            layoutsDir:    viewRoot + "/layouts/",
+            partialsDir:   viewRoot + "/partials/"
+        };
+
+        if (helpersToLoad) {
+            handlebarsConfig.helpers = helpersToLoad;
+        }
+
+
+        that.model.express.set("views", viewRoot);
+
+        var hbs = exphbs.create(handlebarsConfig);
+        that.model.express.engine("handlebars", hbs.engine);
+        that.model.express.set("view engine", "handlebars");
+    }
+    else {
+        console.error("Cannot initialize template handling without a 'config.express.views' option");
+    }
+};
+
+express.startPrivate = function(that, callback) {
+    if (!that.options.config || !that.options.config.express) {
+        console.error("Cannot start express because you have not supplied a 'config' option.");
+        return;
+    }
+
+    that.model.express.set("port", that.options.config.express.port);
+    that.server = that.model.express.listen(that.options.config.express.port, function(){
+        console.log("Express server listening on port " + that.model.express.get("port"));
+
+        console.log("Express started...");
+        that.events.expressStarted.fire();
+
+        if (callback) {
+            callback();
+        }
+    });
+};
+
+express.stopPrivate = function(that, callback) {
```

Comment:
As with the mail server, I collapsed the additional start function into the existing init function.

I renamed the stop function, but it is no longer exposed, and is called as part of the normal "destroy" lifecycle.